### ISSUE TITLE
Remove or adding other plugins gives sentry messages

### ISSUE THREAD

**Issue by mcfarljw** [getsentry/sentry-cordova#66]
Whenever I run `cordova plugin remove cordova-plugin-file` or any other non-sentry plugin while `cordova-sentry` is installed it tells me that sentry was successfully uninstalled. It also displays sentry related wizard messages when running `cordova plugin add cordova-plugin-file`. Sentry should only do stuff when adding or removing it, not other plugins.

**Comment by HazAT** [comment]
Hmm, this shouldn't happen, I suppose you are using the latest version?

**Comment by mcfarljw** [comment]
Yes, I am using the latest version which I believe is `0.10.2`. The specific plugin I was removing at the time was `cordova-sqlite-storage`.

**Comment by mcfarljw** [comment]
```
Saved plugin info for "cordova-plugin-testing" to config.xml
Sentry: running after_plugin_add - set SENTRY_SKIP_WIZARD=true to skip this
```

I'm including a bit of the console log showing that sentry is running when any platform operations happen. It looks like their is a variable that can be set, but I don't think that should be required when managing other plugins.
